### DISCUSSION TITLE
Campaign tutorial using incorrect characters

### DISCUSSION THREAD

**Discussion by DownloadableContent5021** [FakeFishGames/Barotrauma#14778] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When starting a new save, sometimes the order of the starting characters is different. Normally this has no effect on gameplay, but during the campaign tutorial, you're supposed to talk to the engineer while he tells you to how to play, but since you'll start as the engineer, he'll just start talking to himself. This doesn't cause any real issues though, because the tutorial still works perfectly fine.

This only happens sometimes, but the order of the starting crew seems to change each time the game is quit and re-opened. I don't, however, have any other ideas as to what causes this or how else to re-create this issue.

![Screenshot 2024-10-06 at 9 25 36 AM](https://github.com/user-attachments/assets/b6e7e63c-93d1-4acb-b4d2-87372eb1933a)
![Screenshot 2024-10-06 at 9 26 07 AM](https://github.com/user-attachments/assets/0283eca9-9bcd-4456-b8eb-02731adec23c)
[brokentutorial save file.zip](https://github.com/user-attachments/files/17267651/brokentutorial.save.file.zip)

### Reproduction steps

1. Open game and check to see the starting order of the crew in a campaign.
2. If the crew order is normal, quit and reopen the game to try again.
3. If crew order is incorrect, start a new singleplayer campaign with tutorial to see the broken text.

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

MacOS

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly. Fix coming in the patch after the upcoming hotfix update. Closing discussion.
